admin_app.py

```diff
@@ -25,59 +28,90 @@ def make_secret_key():
                      '/account_admin'))
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 db = SQLAlchemy(app)
+
+
+""" Auth0 setup and methods
 """
-JWT token validator and user identity helper
-"""
+AUTH0_CALLBACK_URL = os.getenv('AUTH0_CALLBACK_URL')
+AUTH0_CLIENT_ID = os.getenv('AUTH0_CLIENT_ID')
+AUTH0_CLIENT_SECRET = os.getenv('AUTH0_CLIENT_SECRET')
+AUTH0_DOMAIN = os.getenv('AUTH0_DOMAIN')
+AUTH0_BASE_URL = 'https://' + AUTH0_DOMAIN
+AUTH0_AUDIENCE = os.getenv('AUTH0_AUDIENCE')
```

Comment:
`os.getenv` accepts a default, so this could be `os.getenv('AUTH0_AUDIENCE', AUTH0_BASE_URL + '/userinfo')` if you like.